### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by harshitg00**

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -5,6 +5,9 @@
 /.pnp
 .pnp.js
 
+#Enviornment Variables
```
Comment (prayutsu) [inline frontend/.gitignore]: Add a space after '#' and 'V' should be in lowercase

**Review by prayutsu — CHANGES_REQUESTED** [review]
@harshitg00 PTAL

File: frontend/.gitignore (hunk 1 above)
Comment (harshitg00) [reply frontend/.gitignore]: Done!

**Review by prayutsu — APPROVED** [review]
LGTM!

**Pull request merged**